draft-ietf-tls-tls13.md

```diff
@@ -2531,6 +2531,8 @@ to perform post-handshake authentication {{post-handshake-authentication}}. Serv
 MUST NOT send a post-handshake CertificateRequest to clients which do not
 offer this extension. Servers MUST NOT send this extension.
 
+%%% Post Handshake Authentication Extension
```

Comment:
I think that `%%% Server Parameters Messages` is more appropriate.

(otherwise it would have to be added to the `Protocol Data Structures and Constant Values` section)

Reply:
It's up to @ekr. :-)

Reply:
I concur with @Lekensteyn